addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -280,16 +280,18 @@ export class RelationalModel extends Model {
                 config.orderBy = config.orderBy.filter((order) => order.name !== "__count");
             }
         }
-        if (!config.isMonoRecord && this.root && params.domain) {
+        if (!config.isMonoRecord && params.domain) {
             // always reset the offset to 0 when reloading from above with a domain
             const resetOffset = (config) => {
                 config.offset = 0;
                 for (const group of Object.values(config.groups || {})) {
                     resetOffset(group.list);
                 }
             };
-            resetOffset(config);
-            if (!!config.groupBy.length !== !!currentGroupBy.length) {
+            if (this.root) {
+                resetOffset(config);
+            }
+            if (!!config.groupBy?.length !== !!currentGroupBy?.length) {
```

Comment:
no groupby even if !monorecord?

Reply:
No, it's missing in both list and kanban views

Reply:
weird